lib/assessment.md

```diff
@@ -22,14 +22,29 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+That mean that the web is a service between the client and a server. They both need each other to be a service. The client asks for information from the server and the server provide the data the client needs.
 <!-- your answer ends here -->
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
 <!-- your answer starts here -->
-
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Test Webpage!!!</title>
+  </head>
+  <body>
+    <ul>
+      <li>One</li>
+      <li>Two</li>
+      <li>Three</li>
+    </ul>
+   </body>
+   <footer>
+    <p>Something smells like feet down here...</p>
+  </footer>
```

Comment:
I can see why you'd do this, but it's not correct. `footer`, like `header` (not to be confused with `head`) represents a section of space on the page; consequently, it should go in the `body`.